lightly/active_learning/scorers/classification.py

```diff
@@ -93,10 +94,24 @@ class ScorerClassification(Scorer):
 
     """
     def __init__(self, model_output: Union[np.ndarray, List[List[float]]]):
-        if isinstance(model_output, List):
+        if not isinstance(model_output, np.ndarray):
             model_output = np.array(model_output)
+
+        self.ensure_model_output_is_correct(model_output)
+
         super(ScorerClassification, self).__init__(model_output)
 
+    def ensure_model_output_is_correct(self, model_output: np.ndarray):
```

Comment:
not a huge thing but I'd say typically the name of functions like this is along the lines of `ensure_valid_model_output` and not a full sentence